digital_twins/mass-spring-damper-monitor/README.md

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
There is more explanation needed here. Perhaps this file (https://github.com/INTO-CPS-Association/DTaaS-examples/blob/main/digital_twins/opc-ua-waterplant/README.md) can serve as inspiration. Even this README has been updated at https://github.com/INTO-CPS-Association/DTaaS/blob/feature/distributed-demo/docs/user/examples/mass-spring-damper/README.md. Please copy the same and use it. Please add an updated figure for this example in   https://github.com/INTO-CPS-Association/DTaaS/blob/feature/distributed-demo/docs/user/examples/examples.drawio

You can make this figure a part of the PR.